django_mongodb/fields/auto.py

```diff
@@ -15,6 +15,14 @@ def __init__(self, *args, **kwargs):
         kwargs["db_column"] = "_id"
         super().__init__(*args, **kwargs)
 
+    def deconstruct(self):
+        name, path, args, kwargs = super().deconstruct()
+        if self.db_column == "_id":
+            del kwargs["db_column"]
+        if path.startswith("django_mongodb.fields.auto"):
+            path = path.replace("django_mongodb.fields.auto", "django_mongodb.fields")
```

Comment:
Why do we do this? Just for consistency in path imports or is there a technical reason, too?

Reply:
Similar to https://github.com/django/django/blob/163e72ebbaa84804877f3d1ae212575e479b533b/django/db/models/fields/__init__.py#L646-L657, the fact that our fields are implemented in submodules of `django_mongodb.fields` is an implementation detail that's fine to hide from end users.